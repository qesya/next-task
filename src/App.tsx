import React from 'react';
import { Provider } from 'react-redux';

import store from './redux/store';
import { AppGlobalStyle } from './styles/global';
import Container from './components/atoms/Container';
import ResultList from './components/molecules/ResultList';
import SearchForm from './components/organisms/SearchForm';


function App() {

	return (
		<Provider store={store}>
			<Container column>
				<AppGlobalStyle />
				<SearchForm />
				<ResultList />
			</Container>
		</Provider>
	);
}

export default App;
