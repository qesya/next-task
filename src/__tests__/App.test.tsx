import React from 'react';
import { render } from '@testing-library/react';
import App from '../App';

describe('<App />', () => {
	test('should match snapshot', () => {
		const { asFragment } = render(<App />);
		expect(asFragment()).toMatchSnapshot();
	});
});
