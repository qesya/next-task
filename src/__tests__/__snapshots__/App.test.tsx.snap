// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-bczRLJ kXcazA"
  >
    <div
      class="sc-bczRLJ sc-hKMtZM jNgGJB eYeKUE"
    >
      <input
        class="sc-eCYdqJ bSDTdO"
        placeholder="Search"
        value=""
      />
    </div>
  </div>
</DocumentFragment>
`;
